version: '3.7'
services:
  standalone-eventlog:
    image: standalone-eventlog
    ports:
      - 42008:42008
    environment:
      - ISDOCKER=cat
    networks:
      - praxisbot
    volumes:
      - "../logs:/Praxis/logs"
      - "../credentials:/Praxis/credentials"
      - "../config.py:/Praxis/config.py"
networks:
  praxisbot:
    name: praxisbot